lib/stdlib/doc/src/proc_lib.xml

```diff
@@ -159,6 +159,17 @@
       </desc>
     </func>
 
+    <func>
+      <name name="get_label" arity="1" since="OTP 26.2"/>
```

Comment:
I'm interested to learn if this will get included in the upcoming 26.2? It seems this branch was merged to master instead of maint, but the docs say since 26.2 here and for set_label/1.

Reply:
No master only, we changed our mind and forgot to update the docs, thanks.